sktime/performance_metrics/annotation/metrics.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+"""
+Metrics for evaluating performance of segmentation estimators.
+
+Metrics are suitable for comparing predicted change point sets
+against true change points and quantify the error.
+"""
+
+import numpy as np
+import numpy.typing as npt
+from scipy.spatial.distance import directed_hausdorff
+from sklearn.utils import check_array
+
+__author__ = ["lmmentel"]
+
+
+def annotation_error(
+    true_change_points: npt.ArrayLike, pred_change_points: npt.ArrayLike
+) -> float:
+    """
+    Annotation error measuring discrepancy in the number of change points.
+
+    Parameters
+    ----------
+    true_change_points: array_like
+        Indexes of true change points
+    pred_change_points: array_like
+        Indexes of predicted change points
+
+    Returns
+    -------
+        annotation_error
+    """
+    true_change_points = check_array(true_change_points, ensure_2d=False)
+    pred_change_points = check_array(pred_change_points, ensure_2d=False)
+    return abs(true_change_points.size - pred_change_points.size)
+
+
+def hausdorff_error(
+    true_change_points: np.array, pred_change_points: np.array, symmetric: bool = True
+) -> float:
+    """
+    Hausdorff metric measures how far two subsets of a metric space are from each other.
+
```

Comment:
worth explaining in the docstring which metric space the true and predicted change points live in

Reply:
also worth saying in docstring that you are wrapping scipy function. (do we want user to be able to choose all parameters for scipy function. e.g. it has a seed?)